services/src/test/java/com/jos/dem/jmailer/service/EmailValidatorServiceTest.java

```diff
@@ -22,28 +22,28 @@
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.TestInfo;
-import org.mockito.Mock;
 import org.mockito.MockitoAnnotations;
 
 import java.util.Arrays;
 import java.util.List;
 
 import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.mockito.Mockito.when;
 
 @Slf4j
 class EmailValidatorServiceTest {
 
   private EmailValidatorService emailValidatorService;
-
-  @Mock private EmailProperties emailProperties;
+  private final EmailProperties emailProperties = new EmailProperties();
 
   private List<String> spamTokens = Arrays.asList("one", "two", "three");
+  private List<String> spamNames = Arrays.asList("John", "Edward", "Sebastian");
```

Comment:
You can make both `spamTokens` and `spamNames` final

Reply:
Done, good catch!